CryptoAnalysis/src/main/java/crypto/HeadlessCryptoScanner.java

```diff
@@ -420,6 +455,14 @@ protected boolean providerDetection() {
 		return true;
 	}
 	
+	protected List<String> getIgnorePackageList() {
```

Comment:
Why does this one always return null?

Reply:
It should return `Collections.EMPTY_LIST`. 
also i'm not sure yet if these should be public or protected

Reply:
Replaced null with Collections.<String>emptyList()